src/citrine/informatics/descriptors.py

```diff
@@ -94,6 +94,33 @@ def __init__(self, key: str, threshold: float = 1.0):
         self.threshold = threshold
 
 
+class MolecularStructureDescriptor(Serializable['MolecularStructureDescriptor'], Descriptor):
+    """
+    [ALPHA] Material descriptor for an organic molecule (accepts SMILES and InChI String values).
```

Comment:
According to Mithril we also support IUPAC names

Reply:
Ah, yes. Thank you. I will add that.